src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -36,7 +36,22 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
```

Comment:
If we're allowing partial matches probably don't need to split the sentence up anymore right?
Like just return sentence.contains(preppedWord) (after toLowerCase-ing them)

Reply:
Oh yea! True, didn't consider this part. Thank you! Will edit my code.